apps/src/applab/commands.js

```diff
@@ -487,15 +473,11 @@ applabCommands.penWidth = function(opts) {
   }
 };
 
+// the default color of ctx.strokeStyle and ctx.fillStyle is 'black'
```

Comment:
Not sure what this comment exactly means.  Is it describing what the function will return?

Reply:
I added this comment to explain that the initial color of strokeStyle and fillStyle when the canvas is created is 'black'. But I think I'll go ahead. Thanks!